### PR TITLE
feat: added the authentication pages

### PR DESCRIPTION
**Pull request by pradipgharge**
added: login, signup and signout pages.

### CODE REVIEW

**Comment by iprankurpandey** [comment]
card are overlapping give some margin rest is good

**Review by iprankurpandey — APPROVED** [review]

File: pages/Authentication/login/login.html [hunk 1]
```diff
@@ -0,0 +1,107 @@
...
+        <nav class="navbar container-2">
+            <div>
+                <a href="/index.html">
+                    <span class="header-title">Furnbuy</span>
+                </a>
+            </div>
+            <ul class="navbar-left">
+                <li>
+                    <a href="#">
+                        <span class="header-pills">Products</span>
```
Comment (Ankur9669) [inline pages/Authentication/login/login.html:22]: fix: add the link to products page here

File: pages/Authentication/login/login.css [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+    align-items: center;
+}
+
+.form-body{
+    display: flex;
+    flex-direction: column;
+    gap:12px;
+}
+
+.form-footer{
```
Comment (Shivani-1524) [inline pages/Authentication/login/login.css:28]: could extract this into a utility class and use it wherever needed

File: pages/Authentication/login/login.html [hunk 3]
```diff
@@ -0,0 +1,107 @@
...
+                        <img class="btn-icon-img" src="/Assets/icons/btn-search.svg" alt="search icon">
+                    </button>
+                    <input class="search-bar-input" type="text" placeholder="Search products...">
+                </li>
+            </ul>
+            <ul class="navbar-right">
+                <li>
+                    <a href="#">
+                        <div class="badge bd-sm">
+                            <img class="badge-icon" src="/Assets/icons/btn-heart.svg" alt="heart icon">
```
Comment (Ankur9669) [inline pages/Authentication/login/login.html:36]: fix: add link here

File: pages/Authentication/login/login.html [hunk 4]
```diff
@@ -0,0 +1,107 @@
...
+                    </a>
+                </li>
+            </ul>
+        </nav>
+    </header>
+    <section class="homepage-container">
+        <main class="hero-container container-2">
+            <div class="login-form">
+                <span class="form-header">
+                    <h1>Welcome back !</h1>
```
Comment (Ankur9669) [inline pages/Authentication/login/login.html:63]: Question: Why span and not div?

File: index.html [hunk 5]
```diff
@@ -33,22 +33,22 @@
...
+                            <img class="badge-icon" src="/Assets/icons/btn-profile.svg" alt="profile icon">
                         </div>
                     </a>
                 </li>
                 <li>
-                    <a href="/pages/Cart management/cart.html">
+                    <a href="/pages/Cart/cart.html">
                         <div class="badge bd-sm">
-                            <img class="badge-icon" src="/Assets/icons/btn-cart.svg" alt="icon">
+                            <img class="badge-icon" src="/Assets/icons/btn-cart.svg" alt="cart icon">
```
Comment (mehulsatardekar) [inline index.html:51]: Suggestion, the icon badges on Navbar looks a little bit small you can add some padding on badges to make them bigger

File: pages/Authentication/login/login.html [hunk 6]
```diff
@@ -0,0 +1,107 @@
...
+                        </span>
+                        <span>
+                            <a href="#">
+                                <button class="btn btn-link-primary">Forgot password</button>
+                            </a>
+                        </span>
+                    </div>
+                    <div class="form-btn-container">
+                        <button class="btn btn-solid-black btn-wide">Sign in</button>
+                        <button class="btn btn-icon btn-outline-black btn-wide">
```
Comment (Shivani-1524) [inline pages/Authentication/login/login.html:90]: try to add hover effects for all buttons.

File: pages/Authentication/login/login.html [hunk 7]
```diff
@@ -0,0 +1,107 @@
...
+                </span>
+                <form class="form-body">
+                    <div class="std-input">
+                        <label for="user-email">Email</label>
+                        <input id="user-email" type="text">
+                    </div>
+                    <div class="std-input">
+                        <label for="user-password">Password</label>
+                        <input id="user-password" type="text">
+                    </div>
```
Comment (Ankur9669) [inline pages/Authentication/login/login.html:74]: fix: use type = "password"

**Review by mehulsatardekar — APPROVED** [review]
Rest looks good @pradipgharge you can merge the branch after changes.👍

File: pages/Authentication/login/login.html [hunk 8]
```diff
@@ -0,0 +1,107 @@
...
+                                <button class="btn btn-link-primary">Forgot password</button>
+                            </a>
+                        </span>
+                    </div>
+                    <div class="form-btn-container">
+                        <button class="btn btn-solid-black btn-wide">Sign in</button>
+                        <button class="btn btn-icon btn-outline-black btn-wide">
+                            <img class="btn-icon-img" src="/Assets/icons/google1.svg" alt="google icon">
+                            <span>Sign in with Google</span>
+                        </button>
```
Comment (Ankur9669) [inline pages/Authentication/login/login.html:93]: Suggestion: try to use p here, or just remove the span tag it should work. span is not a semantic element so the screen readers would omit the text written inside it, and here this is a important text, which you will want the user to know about

File: pages/Authentication/signup/signup.html [hunk 9]
```diff
@@ -0,0 +1,120 @@
...
+                        </span>
+                        <span class="input-wrapper">
+                            <label for="second-name">Second name:</label>
+                            <input id="second-name" type="text"/>
+                        </span>
+                    </div>
+                    <div class="std-input">
+                        <label for="user-email">Email</label>
+                        <input id="user-email" type="text">
+                    </div>
```
Comment (Ankur9669) [inline pages/Authentication/signup/signup.html:80]: fix: use type = "email"

File: pages/Authentication/signup/signup.html [hunk 10]
```diff
@@ -0,0 +1,120 @@
...
+                            </label>
+                        </span>
+                    </div>
+                    <div class="form-btn-container">
+                        <button class="btn btn-solid-black btn-wide">Sign up</button>
+                        <span class="text-center">OR</span>
+                        <button class="btn btn-icon btn-outline-black btn-wide">
+                            <img class="btn-icon-img" src="/Assets/icons/google1.svg" alt="google icon">
+                            <span>Sign up with Google</span>
+                        </button>
```
Comment (Ankur9669) [inline pages/Authentication/signup/signup.html:106]: Same suggestion here

**Review by Ankur9669 — CHANGES_REQUESTED** [review]
UI improvement: 
image

You can have hover effects on sign in buttons

image

The gray text is not visible properly.

image


You can write last name instead of second name

File: pages/Authentication/login/login.css [hunk 11]
```diff
@@ -0,0 +1,58 @@
...
+    align-items: center;
+}
+
+.form-btn-container{
+    display: flex;
+    flex-direction: column;
+    padding:8px 0;
+    gap:8px;
+}
+.btn-link-primary{
```
Comment (Shivani-1524) [inline pages/Authentication/login/login.css:46]: on click the button overlaps with the components on top. try to fix that.

File: pages/Authentication/login/login.html [hunk 12]
```diff
@@ -0,0 +1,107 @@
...
+                    <a href="#">
+                        <div class="badge bd-sm">
+                            <img class="badge-icon" src="/Assets/icons/btn-heart.svg" alt="heart icon">
+                            <span class="bd-count">2</span>
+                        </div>
+                    </a>
+                </li>
+                <li>
+                    <a href="/pages/Authentication/login/login.html">
+                        <div>
```
Comment (Shivani-1524) [inline pages/Authentication/login/login.html:43]: image
 try fixing this component - badges could be bigger and the icon sizes could be more consistent

**Review by Shivani-1524 — CHANGES_REQUESTED** [review]

File: pages/Authentication/login/login.html (hunk 1 above)
Comment (pradipgharge) [reply pages/Authentication/login/login.html:22]: Haven't made yet, will add soon

File: pages/Authentication/login/login.html (hunk 4 above)
Comment (pradipgharge) [reply pages/Authentication/login/login.html:63]: Not any specific reason, I mostly used span inside div for small code blocks.

File: pages/Authentication/login/login.html (hunk 12 above)
Comment (pradipgharge) [reply pages/Authentication/login/login.html:43]: Hey, thanks for the suggestion. Actually need to fix this in the component library, then update it in e-com. Shall do after completing P0 components.

File: pages/Authentication/login/login.html (hunk 6 above)
Comment (pradipgharge) [reply pages/Authentication/login/login.html:90]: have added an active pseudo-class on buttons.

File: pages/Authentication/login/login.html (hunk 8 above)
Comment (pradipgharge) [reply pages/Authentication/login/login.html:93]: Done

File: pages/Authentication/signup/signup.html (hunk 9 above)
Comment (pradipgharge) [reply pages/Authentication/signup/signup.html:80]: Done

File: pages/Authentication/signup/signup.html (hunk 10 above)
Comment (pradipgharge) [reply pages/Authentication/signup/signup.html:106]: Done

File: pages/Authentication/login/login.html (hunk 3 above)
Comment (pradipgharge) [reply pages/Authentication/login/login.html:36]: haven't added products page, will do soon

File: index.html (hunk 5 above)
Comment (pradipgharge) [reply index.html:51]: Have to fix that in component first, will do that .

File: pages/Authentication/login/login.html (hunk 7 above)
Comment (pradipgharge) [reply pages/Authentication/login/login.html:74]: Done

File: pages/Authentication/login/login.html (hunk 8 above)
Comment (Shivani-1524) [reply pages/Authentication/login/login.html:93]: > Suggestion: try to use p here, or just remove the span tag it should work. span is not a semantic element so the screen readers would omit the text written inside it, and here this is a important text, which you will want the user to know about

woah i had no idea. thanks for sharing this!

**Review by Shivani-1524 — APPROVED** [review]

**Review by Ankur9669 — APPROVED** [review]

**Pull request merged**